shared/unlock-folders/remote-container.desktop.js

```diff
@@ -12,7 +12,7 @@ const mapDispatchToProps = dispatch => ({
   toPaperKeyInput: () => dispatch(UnlockFoldersGen.createToPaperKeyInput()),
 })
 export default connect(
-  state => state,
+  (state: any) => state,
```

Comment:
flow is correct here, connect assumes the real store but the remote containers use a totally different store shape